gms/impl/src/test/resources/corpGroupESAutocompleteRequest.json

```diff
@@ -7,7 +7,6 @@
   },
   "post_filter" : {
     "bool" : {
-      "disable_coord" : false,
```

Comment:
These are auto-generated anyway?

Reply:
Yes. It's worth calling out in the migration summary doc, I think.